cedar-policy-formatter/tests/trivial_forbid.cedar

```diff
@@ -0,0 +1 @@
+forbid (principal, action, resource);
```

Comment:
is it intended that this is already properly formatted? (Same for the other files)

Reply:
This one is already formatted, but it's not important either way. Tests should probably cover a mix of polices that are formatted and not

Reply:
Can we add some more not-already-formatted tests then?  (Do we have many already?)

Reply:
We should. After this PR it's just a matter of adding a new `.cedar` file to tests.